### ISSUE TITLE
[jvm] Broken behavior: Manifest generation places class-path on a single line preventing the jar from running.

### ISSUE THREAD

**Issue by davidbruce** [HaxeFoundation/haxe#10157]
The manifest file generated from `--jvm` is placing the entire class-path on a single line.  This doesn't become an issue until you have a large set of libraries and it prevents the jar from running.
This does not effect `--java` and works as expected.  

Patching the jar generated by `--jvm` with the one from `--java` allows the jar to work as expected.  

Generated manifest with `--java`:
```
Main-Class: bgfx.Main
Class-Path: lib/lwjgl-bgfx-3.2.4-SNAPSHOT.jar 
 lib/lwjgl-spvc-3.2.4-SNAPSHOT.jar 
 lib/lwjgl-assimp-3.2.4-SNAPSHOT.jar 
 lib/lwjgl-shaderc-3.2.4-SNAPSHOT.jar 
 lib/lwjgl-glfw-3.2.4-SNAPSHOT.jar 
 lib/lwjgl-vma-3.2.4-SNAPSHOT.jar 
 lib/lwjgl-3.2.4-SNAPSHOT.jar 
 lib/lwjgl-vulkan-3.2.4-SNAPSHOT.jar 
 lib/lwjgl-opengl-3.2.4-SNAPSHOT.jar 
 lib/lwjgl-3.2.4-SNAPSHOT-natives-macos.jar 
 lib/lwjgl-shaderc-3.2.4-SNAPSHOT-natives-macos.jar 
 lib/lwjgl-opengl-3.2.4-SNAPSHOT-natives-macos.jar 
 lib/lwjgl-glfw-3.2.4-SNAPSHOT-natives-macos.jar 
 lib/lwjgl-assimp-3.2.4-SNAPSHOT-natives-macos.jar 
 lib/lwjgl-bgfx-3.2.4-SNAPSHOT-natives-macos.jar 
 lib/lwjgl-spvc-3.2.4-SNAPSHOT-natives-macos.jar 
 lib/lwjgl-vulkan-3.2.4-SNAPSHOT-natives-macos.jar 
 lib/lwjgl-vma-3.2.4-SNAPSHOT-natives-macos.jar 
 lib/org.eclipse.swtchart-d61f136.jar 
 lib/org.eclipse.swt-3.115.0.jar 
 lib/org.eclipse.swtchart.extensions-d61f136.jar 
 lib/org.eclipse.swt.cocoa.macosx.x86_64-3.115.0.jar 
```

Generated manifest with `--jvm`:
```
Manifest-Version: 1.0
Class-Path: lib/lwjgl-bgfx-3.2.4-SNAPSHOT.jar lib/lwjgl-spvc-3.2.4-SNAPSHOT.jar lib/lwjgl-assimp-3.2.4-SNAPSHOT.jar lib/lwjgl-shaderc-3.2.4-SNAPSHOT.jar lib/lwjgl-glfw-3.2.4-SNAPSHOT.jar lib/lwjgl-vma-3.2.4-SNAPSHOT.jar lib/lwjgl-3.2.4-SNAPSHOT.jar lib/lwjgl-vulkan-3.2.4-SNAPSHOT.jar lib/lwjgl-opengl-3.2.4-SNAPSHOT.jar lib/lwjgl-3.2.4-SNAPSHOT-natives-macos.jar lib/lwjgl-shaderc-3.2.4-SNAPSHOT-natives-macos.jar lib/lwjgl-opengl-3.2.4-SNAPSHOT-natives-macos.jar lib/lwjgl-glfw-3.2.4-SNAPSHOT-natives-macos.jar lib/lwjgl-assimp-3.2.4-SNAPSHOT-natives-macos.jar lib/lwjgl-bgfx-3.2.4-SNAPSHOT-natives-macos.jar lib/lwjgl-spvc-3.2.4-SNAPSHOT-natives-macos.jar lib/lwjgl-vulkan-3.2.4-SNAPSHOT-natives-macos.jar lib/lwjgl-vma-3.2.4-SNAPSHOT-natives-macos.jar lib/org.eclipse.swtchart-d61f136.jar lib/org.eclipse.swt-3.115.0.jar lib/org.eclipse.swtchart.extensions-d61f136.jar lib/org.eclipse.swt.cocoa.macosx.x86_64-3.115.0.jar
Created-By: Haxe (Haxe Foundation)
Main-Class: bgfx.Main
```

**Comment by Simn** [comment]
Huh... I mean, we can change this, but I'm surprised this causes problems...

**Comment by davidbruce** [comment]
Fixed with https://github.com/HaxeFoundation/haxe/pull/10160

**Comment by davidbruce** [comment]
Actually, I don't know the process you guys follow.  Should I have close this issue?

**Comment by Simn** [comment]
Yeah, usually PRs mention something like "closes #10157" in the description anyway and then GitHub takes care of it, but this works too. Thanks!
